routes.yaml

```diff
@@ -0,0 +1,5 @@
+app_name: limited-partnerships-api
+group: api
+weight: 100
+routes:
+  1: ^/transactions/(.*)/limited-partnerships/*
```

Comment:
@mikemoore24 's service design specifies the end-points with `limited-partnership` at the end not pluralised, so I think you need to remove the 's'.

See https://companieshouse.atlassian.net/wiki/spaces/Arch/pages/4722491429/LP+Service+Design

Reply:
Probably not relevant as this is a Mesos related file, so will resolve this comment once file is removed.

Reply:
Removed the "s".

Reply:
Removed the whole file.